File: src/main/java/com/craftmanship/ErrorInfo.java
package com.craftmanship;

public class ErrorInfo {
	private Integer row;
	private String detailMessage;

	public ErrorInfo(Integer row, String detailMessage) {
		super();
		this.row = row;
		this.detailMessage = detailMessage;
	}

	public String message() {
		return String.format("Invalid data in row %d (%s)", row, detailMessage);
	}
}

File: csharp/DataValidator/ErrorInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataValidatorProject
{
    public class ErrorInfo: Object
    {
		private int row;
		private string detailMessage;

		public ErrorInfo(int row, string detailMessage)
		{
			this.row = row;
			this.detailMessage = detailMessage;
		}

		public string message()
		{
			return String.Format("Invalid data in row %d (%s)", row, detailMessage);
		}
	}
}
